templates/about/index.html.twig

```diff
@@ -27,6 +27,27 @@
             </div>
         </div>
     </div>
-
+    <section class="missions my-5">
+        <div class="row">
+            <h2 class="text-center">Nos missions</h2>
+        </div>
+        <div class="mission">
+            {% for mission in missions %}
+            <div
+                class="row flex-column text-center flex-xxl-row align-items-center justify-content-center my-5 {{ loop.index % 2 != 0 ? 'flex-xxl-row-reverse' }}">
```

Comment:
pour info, en twig tu peux faire un "is even" ou "is odd" plutot

Reply:
effectivement